mod3hw/bonusHW/css/style_flexbox.css

```diff
@@ -0,0 +1,99 @@
+body {
+    margin: 0;
+    font-family: Arial, Helvetica, sans-serif;
+    color: white;
+}
+
+
+main {
+    background-image: url(../img/background.png);
+    margin: auto;
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 100vh;
+    width: 960px;
+}
+
+.left-logo {
+    display: flex;
+    flex-direction: column;
+}
+
+.left-logo_sloghan {
+    margin-top: -75px;
+    margin-left: -90px;
+}
+
+.left-logo_palm {
+    margin-left: 50px;
+    margin-top: 50px;
+    height: 147px;
+    width: 125px;
+}
+
+.logo-contacts {
+    display: flex;
+    justify-content: space-between;
+    margin-left: 155px;
+    font-size: 10px;
+}
+
+.contacts_list {
+    margin-bottom: 0;
+    margin-top: 147px;
+    margin-right: 80px;
+    height: fit-content;
+    list-style: none;
+}
+
+.contacts_list a, .main-content__pictures a {
+    text-decoration: none;
+    color: white;
+}
+
+.contacts_list a:visited, .main-content__pictures a:visited {
+    color: white;
+}
+
+.asia-travel {
+    margin-bottom: 0;
+}
+
+.main-content-pictures__list {
+    padding: 0;
+    display: flex;
+    flex-wrap: wrap;
+    list-style: none;
+    justify-content: center;
+}
+
+.main-content-pictures__list li {
+    margin: 0 6px 30px;
+}
+
+.holiday-place {
+    margin: 0;
+}
+
+.holiday-place figcaption{
+    font-size: 12px;
+    text-align: center;
+}
+
+.main-content__pictures img {
+    border-style: solid;
+    border-width: 5px 5px 5px 5px;
+    -moz-border-image: url(../img/ramka.png) 7 7 7 7 repeat repeat;
```

Comment:
since you're using flexbox you aren't supporting old browsers so you don't need to add browsers prefix here. In the real project `autoprefixer` plugin will add them automatically following browser support rules that you've specified.
When I develop I usually reference to the https://caniuse.com